### PR TITLE
Adding unit tests for sample-code-java

### PR DESCRIPTION
**Pull request by EdgarScott**
I've analysed your codebase and noticed that it is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look further.

### CODE REVIEW

**Comment by EdgarScott** [comment]
Starting with a clean branch seemed the easiest way to get rid of all those changed files.

**Comment by EdgarScott** [comment]
@paulkennethkent this is the repo which Cover did not generate any good tests for. We thought it was because Cover doesn't test system.out

Do you want to submit this PR externally anyway, as it was a user request?

File: src/test/java/net/authorize/sample/Sha512/ComputeTransHashSHA2Test.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package net.authorize.sample.Sha512;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ComputeTransHashSHA2Test {
```
Comment (paulkennethkent) [inline src/test/java/net/authorize/sample/Sha512/ComputeTransHashSHA2Test.java:8]: Please submit this class only. Remove all unnecessary dependencies.

**Review by paulkennethkent — CHANGES_REQUESTED** [review]
So comment above.

**Pull request closed without merging**